src/UseCases/AddDonation/AddDonationResponse.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\UseCases\AddDonation;
+
+use WMDE\Fundraising\Frontend\Domain\Model\Donation;
+use WMDE\Fundraising\Frontend\ResponseModel\ValidationResponse;
+
+/**
+ * @license GNU GPL v2+
+ * @author Kai Nissen < kai.nissen@wikimedia.de >
+ */
+class AddDonationResponse extends ValidationResponse {
+
+	private $donation;
+
+	public function __construct( Donation $donation ) {
+		$this->donation = $donation;
+		parent::__construct();
+	}
+
+	public function getDonation(): Donation {
```

Comment:
I'm fine (though not very happy) with returning the domain object here, though then we should be real careful with not using domain logic in the controller or putting controller or presentation related concerns into the domain object.